### ISSUE TITLE
FR: Add lock local-enable cmd OR add hint to doc / --help

### ISSUE THREAD

**Issue by Zocker1999NET** [tailscale/tailscale#10849]
### What are you trying to do?

TL;DR: make it more accessible that `tailscale down && tailscale up` reverts `tailscale lock local-disable`

I temporarily disabled Tailscale Lock on one of my devices using `tailscale lock local-disable` to test if it would help me in a certain situation. In my case, this was a device with one trusted signing key.

Afterwards, I wanted to re-enable it, but I could not find any documentation on how to re-enable tailscale lock. I searched
- `tailscale lock --help`
- `tailscale lock local-disable --help`
- `tailscale lock status`
- https://tailscale.com/kb/1226/tailnet-lock
- https://tailscale.com/kb/1243/tailscale-lock#lock-local-disable
- GitHub Issues (`"lock" "local-disable"`)

`tailscale lock init` fails as well (making sense as Lock was already established for my tailnet). I became somewhat desperate because I do not wanted to re-auth my device until I figured out that `tailscale down && tailscale up` re-enables tailscale lock.

### How should we solve this?

- add to docs (esp. `--help`) that `local-disable` does not sustain a Tailscale restart (e.g. `down && up`)
- add same info (how to re-enable Tailscale lock) to `tailscale lock status`
- for improved usability, add `tailscale lock local-enable` command (if possible / feasible)

### What is the impact of not solving this?

Disturbed users who might file support requests because they might not get the idea that the can re-enable

### Anything else?

For me, it was not obvious that `tailscale down && tailscale up` does revert `local-disable`, esp. because the doc does mention that executing `tailscale lock local-disable` might be used to effectively disable Tailscale Lock when missing
